db/utils.py

```
@@ -51,4 +51,21 @@ async def get_file_by_id(file_id: str):
+    from bson import ObjectId
+    db = await get_db()
+
+    try:
+        result = await db.files.delete_one({"_id": ObjectId(file_id)})
+
+        if result.deleted_count == 1:
+            return {"message": "File deleted successfully"}
+        else:
+            return {"message": "File not found"}
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)